tests/merge_base_class_test.py

```diff
@@ -0,0 +1,32 @@
+import ipaddress
+import pandas as pd
+from kartograf.merge import BaseNetworkIndex
+
+
+def test_base_dict_create():
+    base = BaseNetworkIndex()
+    state = base.get()
+    assert state == {4: {}, 6: {}}
+
+
+def test_base_dict_update():
+    base = BaseNetworkIndex()
+    state = base.get()
+    network = "10.10.0.0/16"
+    base.update(network)
+    assert state[4][10]
```

Comment:
Maybe assert here on the actual value we expect?

Reply:
And similar to what I wrote above the test could work with adding the network and then checking inclusion without looking at the internals.